contracts/liquidityMining/KyberFairLaunch.sol

```diff
@@ -376,7 +377,7 @@ contract KyberFairLaunch is IKyberFairLaunch, PermissionAdmin, ReentrancyGuard {
   function updatePoolRewards(uint256 _pid) public override {
     PoolInfo storage pool = poolInfo[_pid];
     uint32 lastAccountedBlock = _lastAccountedRewardBlock(_pid);
```

Comment:
Can help edit the comment for `_lastAccountedRewardBlock` function, should be `returns last accounted reward block, either the current block number or the endBlock of the pool` (of -> or)